Devfile.yaml

```diff
@@ -18,9 +18,12 @@ tools:
       - name: spring-boot
         port: 8080
     mountProjectsSources: true
-  - name: petclinic-app
+  - name: mysql
     type: kubernetes
     local: petclinic.yaml
+    selector:
```

Comment:
If k8s recommended labels are used in`petclinic.yaml`:

```yaml
labels:
  app.kubernetes.io/name: db
  app.kubernetes.io/component: database
  app.kubernetes.io/part-of: petclinic
```

we could filter out one specific component without explicitly introducing a `selector`: 
 ```yaml
tools:
  - type: kubernetes
    local: afile.yaml
    name: ...
    component: ... 
    part-of: ...
```

That's less verbose but may be less straightforward too. @gorkem what do you think? Is that what you were thinking about?

Reply:
I do not think the shorthand will work (although it looks much organized)  because kubernetes label recommendations are suggestions and we can not rely on their existence. Somebody may use different labels easily. The way @sleshchenko has it right now is what I had in mind.